client/state/jetpack-connect/test/selectors.js

```diff
@@ -354,6 +355,42 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#isRedirectingToWpAdmin()', () => {
```

Comment:
I think you forgot to test what happens if the `state` does't include `jetpackConnect` and/or `jetpackConnectAuthorize` objects. It should probably return `null`.